api/src/main/java/io/opencensus/tags/Tag.java

```diff
@@ -28,18 +29,39 @@
 @AutoValue
 public abstract class Tag {
 
+  private static final TagMetadata METADATA_UNLIMITED_PROPAGATION =
+      TagMetadata.create(TagTtl.UNLIMITED_PROPAGATION);
+
   Tag() {}
 
   /**
    * Creates a {@code Tag} from the given key and value.
    *
+   * <p>This is equivalent to calling {@code create(key, value,
+   * TagMetadata.create(TagTtl.UNLIMITED_PROPAGATION))}.
+   *
    * @param key the tag key.
    * @param value the tag value.
    * @return a {@code Tag} with the given key and value.
    * @since 0.8
+   * @deprecated in favor of {@link #create(TagKey, TagValue, TagMetadata)}.
    */
+  @Deprecated
   public static Tag create(TagKey key, TagValue value) {
-    return new AutoValue_Tag(key, value);
+    return create(key, value, METADATA_UNLIMITED_PROPAGATION);
```

Comment:
For backwards-compatibility, `Tag`s created with this method will still be propagating, though this method is marked as deprecated so there will be warnings when compiling.